git-annex-remote-rclone

```diff
@@ -1,7 +1,5 @@
-#!/bin/bash
```

Comment:
executable permissions should not be stripped of this file AFAIK

Reply:
No they shouldn't. Excellent spotting!